### ISSUE TITLE
Stub installers no longer working since they changes in bug 1372260.

### ISSUE THREAD

**Issue by MihaiTabara** [mozilla/mozdownload#440]
(mozdownload) mihaitabara@mozspace:[master]~/work/mozilla/clones/git/mozdownload$ mozdownload --version=latest-beta --platform=win32 --stub
Traceback (most recent call last):
....
mozdownload.errors.NotFoundError: Binary not found in folder: https://archive.mozilla.org/pub/firefox/releases/55.0b12/win32/en-US/

Sounds like https://github.com/mozilla/mozdownload/blob/master/mozdownload/scraper.py#L499 needs a bit of tweaking. This is due to the changes that took place in https://bugzilla.mozilla.org/show_bug.cgi?id=1372260.

tl;dr:
beta/release: "Firefox Setup Stub 55.0b11.exe" => "Firefox Installer.exe"
nightly: "firefox-56.0a1.en-US.win32.installer-stub.exe" => "Firefox%20Installer.en-US.exe"

**Comment by whimboo** [comment]
@MihaiTabara I don't understand the `tl;dr` part of your comment. When I check latest nightly build folders on central I see `firefox-57.0a1.en-US.win32.installer-stub.exe` but not `Firefox%20Installer.en-US.exe`. Can you please clarify?

**Comment by whimboo** [comment]
Also your beta/release is unclear to me. Here links to the build directories:

https://archive.mozilla.org/pub/firefox/nightly/2017/08/2017-08-02-10-03-02-mozilla-central/
http://archive.mozilla.org/pub/firefox/candidates/55.0b9-candidates/build2/win32/en-US/

**Comment by whimboo** [comment]
My question has been solved given that I missed that the stub installers have a prefix of `Firefox`  now, which means those files are placed at the very top of each folder now. I was looking under win32 and win64 builds.

I will work on this now.
